### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN014.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#60] [General]
# https://blog.3place.cn/3place/lmyn/LMYN014.html

https://blog.3place.cn/3place/lmyn/LMYN014.html

**Comment by picklecai** [comment]
吠陀，意为明、知识，它是印度最古老的文献材料和文体形式，主要文体是赞美诗、祈祷文和咒语，是印度人世代口口相传、长年累月结集而成的。 “吠陀”的意思是“知识”“启示”的意思。"吠陀"用古梵文写成，是印度宗教﹑哲学及文学之基础。

**Comment by picklecai** [comment]
婆罗门不停地吃喝，
无依无靠的人吃喝，
那些苦行者们吃喝，
那些沙门也在吃喝。

年老的、有病的、
妇女们、小孩子，
他们都吃呀喝呀，
永远也不停止。 

犹如过年，谁不喜欢？
大概，人人满足的愉悦，会传递给神，神就愿意降幅了。

**Comment by picklecai** [comment]
在仪式进行的中间，
婆罗门的辩论多种多样；
他们都能言善辩又聪明，
他们都想相互压倒对方。

除了吃喝，还要清谈。这个，大概是给神看的人类的精神食粮。
刚才是：吃饱了，满足了，请赐福吧。
现在是：谈爽了，满足了，请赐福吧。

**Comment by picklecai** [comment]
参加国王大祭的婆罗门/没有人不精通六分支，没有人不忠诚，不多闻，没有人不擅长言辞。 

三条标准：心忠诚；知识广；口才好；

**Comment by picklecai** [comment]
还拴着十车王的马宝。

马宝：犁牛之子骍且角，虽欲勿用，山川其舍诸？

**Comment by picklecai** [comment]
我们不愿要求土地，因为保卫它我们无能。大地之主！我们经常/沉湎于自己学习之中。至于对我们的报酬，陛下给点什么都行。”

我们只愿意卷学习。
